index.html

```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="index.css" />
+    <script src="index.js" defer></script>
     <title>Juank's portifolio</title>
   </head>
   <body>
```

Comment:
- [ ] [OPTIONAL]
 To enhance the UX, kindly ensure that the _hamburger_ and the _close_ button are in the same position when toggled.

Reply:
Hello @rloterh ,

Thank you so much for your feedback. Could you explain in more detail when you say "kindly ensure that the hamburger and the close button are in the same position when toggled"?.

Thanks in advance!

Reply:
@J-C-S-V 

Let's say, when you click the _hamburger menu_ and it disappears, the **close buttons** should be within that same spot where the mouse pointer clicked the **hamburger menu**. In other words, the _hamburger menu_ and the _close button_ should be on the same _x_ and _y_ coordinates.

Cheers! 😃